NumRush.py

```diff
@@ -38,7 +38,7 @@ def randomOperation(maxi):
         return 2, x * y, x, y
     elif (pick == 3):
         x = x * y
-        return 3, x / y, x, y
+        return 3, x // y, x, y
```

Comment:
Did you confirm if this is supposed to be int division?

Reply:
@chimosky Yes, as `x` and `y` both are integers, In Python2, dividing two ints uses integer division

In Python3, dividing two ints results in a float, but using // acts as integer division.

Reply:
Yeah I checked and it was an int hence I resolved the conversation.